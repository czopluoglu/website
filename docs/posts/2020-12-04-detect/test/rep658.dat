00001100001000000110
00011011111110100111
11010111101010111011
11111100101001110011
11101101111010000100
00010011001010100100
01111000001010111011
11100111011111011111
11001011001111111111
11011101011011100111
01101100010100101010
00001100010000000011
01111110110011101111
11110101111010101110
11110111110111001110
10010110001100000010
10111111000111111111
11110111111111001110
11110000010110111110
10100100011010100001
01100110000010000011
11100101011000010010
11001011011000101010
11011101101110010010
01001000101011011010
00110111001110010010
01101100001011100100
11101101101111100101
10100101001101001110
00110100011001010010
01000000111010100010
00110000001010000010
11001111000010010101
11111110011010110111
11000011101010010001
10110010011110001111
10011011101111101111
00000101010110110011
10010011111110110111
00101101111111111110
11100001100110011101
11000011111111011010
11010110100110100010
01011101111000111110
10011101001010001010
11001110111010111100
11010101010111101111
10110001001001111110
00100111101110111011
00000000001111111110
11100111111001000010
10111110111111110111
11101110111111001100
01001101101000011100
10101100001111100111
10110111111111111111
00000000001000101110
10001110101111100111
10000111100000100010
11011001110001111110
00011000000000000111
00010000000111100111
01011101110000010000
00000001001110101111
10101011111110100001
00000001100011000110
10110100101010001111
10111101111101100110
10001000101111111111
11000101111110100111
10110110111001010010
11101111101010011111
01001001101010011111
11010000011100110011
10011101000111001010
11000000011011111110
11110111101110001111
10000001001000000010
01101011011010100111
10001001001100111001
00000000011000001010
11110110111100000110
00000011111010110010
01000100101100100110
00000000101101100000
00000000010000110110
10001011100111101111
00011100111110101001
00001010111110110011
10111111111111111001
11111101111111110111
01100111111110010011
01001010110000110001
10011111100111101111
11110111111011110110
11010111011000101110
11001001101011101111
11000111100110111110
10100111100000010110
00010001011000011010
00101101011000111111
11010001100000001010
11011001101100101111
11110111010110111110
10011100001110100001
00100111101001101100
01000001011111001110
11101000001111011100
10111101111110101111
00001001111111111111
00000011111110111110
10000000101110000010
00011000101111001011
11010101011111110111
00111011111100111101
10111101111001110101
01011001101010100010
11010011001010101110
01001101111010101010
00000000001101010110
11111110001010100101
00000000001101101111
00101010111000101101
11100001001111011111
10101111110110101111
01100101001111001111
00011101111001111000
00101100101100010111
01100111111000101110
11111111111111110111
10000101010011100111
01111111101110100110
10010001111110101111
10111001110100100100
10001100011001101011
11111111111111111110
11011111111011110111
11111111111111101010
01100010010111100110
10011101011000101011
10000111011100010101
01101000011010100001
00100011010000100101
10111101111010000111
01011001111100010111
10110001100111101111
10000101011000100000
10100000101110101110
11000000001000000011
01110001111111001011
00000001001000001111
01010101100011010101
10001111011001111111
00000001011101111011
10111101111011001111
10011001101010100000
01010110011110100010
11101011011010111001
11011011000010100101
10010111010011111111
10111111111101101111
00000111001111101111
01100100111111011111
00100111010100101111
00001001100111000111
10000101101111001111
10111111111100010011
10110101000101101100
11011101111011101111
10000011001111101011
10010110000001101011
01001011110000000000
11011100101101101100
00101011001001101010
01010001111100001000
00101001101100111111
11001000101010001110
11100000101000000110
10001011011001101111
01000011001111001100
00100101011010100011
10111100011101110011
01010010111101011011
10010011111101101001
01001001010010001101
10000011011110111111
10111101010110110111
00100110001010101011
01111101001000000110
11110011101000100110
11000001010101010001
00000100111100101100
00000001011110001011
11011101111111111101
00011000001011110010
01000000001000100110
00001001010000100101
01100101000110111110
10100000111000010011
10001010101100101001
11111010101110010000
00010000000010101000
11000100101111000100
00000011000111101111
10100110111110111101
11111100111110011001
01101100010101111011
10111101001110001110
01101000000010111011
10000001101110111110
01001110110011000111
00010111111111001101
10010001010100011110
00111001101011000010
00010111101001000111
00001000111111111010
10110101000001011001
10110011110101011111
11001001001010011000
00111111101010001110
11100111101111111111
10001110011000100010
10000101111000000001
00100100111100100001
11111011101110001111
01001001111100001111
00000101110010110111
01010011010000000100
01001111100111111111
10111011101111011011
00000011001001111111
01101000111111111111
00001101001101001110
01010110111110000000
01001011101110000101
01001000001110101110
00000010001001100001
11000001001111111001
01100101001010001110
11011011111010101011
00001000000011001111
10110111110011111110
01010100111110011111
00101011010001001011
10111101101100101110
11111111001111111010
10111111111100010010
11111111011110011111
10100011101111111110
10010000011000010010
