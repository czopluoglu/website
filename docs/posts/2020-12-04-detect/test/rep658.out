-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep658.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.80

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.86
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.8136
                   IDN index value:   0.7211
                           Ratio r:   0.6500

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    1    1    2    1    1    3    2    1
    2    3    3    3    3    4    3    3    3    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    3    4    6    7   10

   -----------CLUSTER  2-------------

    2    5    9   11

   -----------CLUSTER  3-------------

    8   12   13   14   15   17   18   19   20

   -----------CLUSTER  4-------------

   16
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++++----++---------
   +d++++---+------++-+
   ++d+-+-++-+------+--
   +++d+++-+-------+---
   ++-+d+--+-+-+-+---+-
   -++++d+-+-+---+---++
   ---+-+d++---+---+--+
   --+---+d++----+---+-
   --++++++d++++-------
   ++-----++d-++--+-+--
   +-+-++--+-d----++-+-
   --------++-d++-+++++
   ----+-+-++-+d++++++-
   -----------++d++++++
   ----++-+----++d++-+-
   ---------++++++d++++
   -+-+--+---++++++d+--
   -++------+-+++-++d-+
   ----++-+--++++++--d+
   -+---++----+-+-+-++d
 
 No cross validation for this DETECT run
